### PR TITLE
Seeg eeg seperated in butterfly plot

### PR DESCRIPTION
**Pull request by agramfort**
taking over #4805

### CODE REVIEW

**Comment by larsoner** [comment]
Looks reasonable to me assuming it solves the original problem

**Comment by agramfort** [comment]
@stfnrpplngr ok for you?

**Comment by agramfort** [comment]
ok let's say it was ok... somebody will complain if it was not...

**Comment by stfnrpplngr** [comment]
Of course, I am ok with that.

File: mne/viz/utils.py [hunk 1]
```diff
@@ -930,9 +930,9 @@ def _mouse_click(event, params):
 
 def _handle_topomap_bads(ch_name, params):
     """Color channels in selection topomap when selecting bads."""
-    for type in ('mag', 'grad', 'eeg', 'seeg', 'hbo', 'hbr'):
-        if type in params['types']:
-            types = np.where(np.array(params['types']) == type)[0]
+    for t in _DATA_CH_TYPES_SPLIT:
```
Comment (mmagnuski) [inline mne/viz/utils.py]: I thought we wanted to avoid one-letter vars ;)

**Pull request merged**